downloadBookImages.sh

```diff
@@ -5,8 +5,20 @@ height=$(($width*2))
 
 rm -f src/main/resources/com/vaadin/exampledata/BookImageBackground.txt
 for baseurl in `cat src/main/resources/com/vaadin/exampledata/BookImageBackground.src`;
-do 
+do
   url="$baseurl?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=$width&h=$height&q=80"
   echo "Downloading $url"
   (echo -n "data:image/png;base64," ; curl -s "$url"|base64  ) >>  src/main/resources/com/vaadin/exampledata/BookImageBackground.txt
 done
+
+rm -f src/main/resources/com/vaadin/exampledata/BookImageFiles/*
+rm -f src/main/resources/com/vaadin/exampledata/BookImageFileNames.txt
+for baseurl in `cat src/main/resources/com/vaadin/exampledata/BookImageBackground.src`;
+do
+  [[ "$baseurl" =~ [^\/]+$ ]]
+  filename="${BASH_REMATCH[0]}.jpg"
+  url="$baseurl?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=$width&h=$height&q=80&fm=jpg"
+  echo "Downloading $url"
+  curl "$url" --output "src/main/resources/com/vaadin/exampledata/BookImageFiles/$filename"
```

Comment:
If we store the images, then we do not store the base64 encoded version separately but instead we can delete `BookImageBackground.txt` and do the base64 encoding when needed? This would remove the close to 1MB file

Reply:
Added in a new commit